Failures in stcal tests with dev gwcs

stcal tests with dev gwcs show several errors:
https://github.com/spacetelescope/stcal/actions/runs/22045592646/job/63693714793
```
 FAILED tests/outlier_detection/test_utils.py::test_gwcs_blot - gwcs.utils.CoordinateFrameError: An input_frame must be specified if forward_transform is a model.
FAILED tests/outlier_detection/test_utils.py::test_gwcs_blot_fillval[0.0] - gwcs.utils.CoordinateFrameError: An input_frame must be specified if forward_transform is a model.
FAILED tests/outlier_detection/test_utils.py::test_gwcs_blot_fillval[nan] - gwcs.utils.CoordinateFrameError: An input_frame must be specified if forward_transform is a model.
FAILED tests/test_alignment.py::test_calc_pixmap - gwcs.utils.CoordinateFrameError: An input_frame must be specified if forward_transform is a model.
```

Would you open a PR to fix these?

Also it seems useful to add stcal to the downstream testing here. Would you like a separate issue for that?

--- a/.github/workflows/downstream.yml
+++ b/.github/workflows/downstream.yml
@@ -48,7 +48,7 @@ jobs:
       cache-path: ${{ needs.environment.outputs.data_path }}/crds
       cache-key: crds-${{ needs.crds_contexts.outputs.jwst }}
       envs: |
-        - linux: py311-test-jwst
+        - linux: py313-test-jwst
 
   romancal:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@99401c364fa51c9c507d3cd6d272049278ac0b2c # v2.4.0
@@ -62,7 +62,7 @@ jobs:
       cache-path: ${{ needs.environment.outputs.data_path }}/crds
       cache-key: crds-${{ needs.crds_contexts.outputs.jwst }}
       envs: |
-        - linux: py311-test-romancal
+        - linux: py313-test-romancal
 
   romanisim_data:
     needs: [environment]
@@ -91,7 +91,20 @@ jobs:
       cache-restore-keys: |
         ${{ needs.romanisim_data.outputs.cache_key }}
       envs: |
-        - linux: py311-test-romanisim-xdist
+        - linux: py313-test-romanisim-xdist
+  stcal:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@99401c364fa51c9c507d3cd6d272049278ac0b2c # v2.4.0
+    needs: [environment, crds_contexts]
+    with:
+      setenv: |
+        CRDS_PATH: ${{ needs.environment.outputs.data_path }}/crds
+        CRDS_SERVER_URL: https://jwst-crds.stsci.edu
+        CRDS_CLIENT_RETRY_COUNT: 3
+        CRDS_CLIENT_RETRY_DELAY_SECONDS: 20
+      cache-path: ${{ needs.environment.outputs.data_path }}/crds
+      cache-key: crds-${{ needs.crds_contexts.outputs.jwst }}
+      envs: |
+        - linux: stcal
 
   astropy:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@99401c364fa51c9c507d3cd6d272049278ac0b2c # v2.4.0
@@ -99,7 +112,7 @@ jobs:
     with:
       submodules: false
       # Any env name which does not start with `pyXY` will use this Python version.
-      default_python: "3.11"
+      default_python: "3.13"
       envs: |
         - linux: specutils
 
@@ -109,7 +122,7 @@ jobs:
     with:
       submodules: false
       # Any env name which does not start with `pyXY` will use this Python version.
-      default_python: "3.11"
+      default_python: "3.13"
       envs: |
         - linux: ndcube
         - linux: dkist

--- a/tox.ini
+++ b/tox.ini
@@ -109,3 +109,18 @@ commands =
     xdist: -n auto \
     cov: --cov=. --cov-config=pyproject.toml --cov-report=term-missing --cov-report=xml \
     {posargs}
+
+[testenv:stcal]
+change_dir = {env_tmp_dir}
+allowlist_externals =
+    git
+    bash
+extras =
+commands_pre =
+    bash -c "uv pip freeze -q | grep 'gwcs @' > {env_tmp_dir}/requirements.txt"
+    git clone https://github.com/spacetelescope/stcal.git
+    pip install -e stcal[test]
+    pip install -r {env_tmp_dir}/requirements.txt
+    pip freeze
+commands =
+    pytest stcal

